isam-citation-style-author-date.csl

```diff
@@ -0,0 +1,3631 @@
+    <link href="https://www.isam.org.tr/uploads/67f7a5f5216f9.pdf" rel="documentation"/>
+    <author>
+      <name>Hüseyin Mertoğlu</name>
+      <uri>https://orcid.org/0009-0003-6734-2048</uri>
+      <email>huseyin.mertoglu@isam.org.tr</email>
+    </author>
+    <contributor>
+      <name>Taha Yasin Tan</name>
+      <email>tahayasin.tan@isam.org.tr</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <category field="theology"/>
+    <category field="history"/>
+    <category field="humanities"/>
+    <summary>İSAM Citation Style is a study in which the accumulation of İSAM’s knowledge is reshaped, interpreted and new standards are produced according to the special needs of İSAM, taking into account the standards of international systems developed to prove knowledge in academic texts. It is prepared according to the needs of İSAM and can be used in all academic studies and publications in the fields of humanities, theology and social sciences. The style was designed to generate the citation outputs requested by İSAM (Center for Islamic Studies) from external submitters of articles and texts for a wide range of work types. The official guide, which includes citation instructions, is available at https://www.isam.org.tr/uploads/67f7a5f5216f9.pdf, and a detailed online user guide is available at https://isamveri.org/atif/. Currently, it is accessible only to limited users via login. The online guide explains how to enter works into Zotero with examples and Zotero screenshots. As can be seen from the guides, citation formatting is requested for numerous work types. Therefore, the number of lines of code in the CSL file is close to 4 thousands. For example, while there is only one artwork type in Zotero, there are seven in the guide. To prevent these types, which require different styles, from negatively impacting each other, these subtypes are managed in the CSL lines using variables in the Extra field. All style outputs have been observed to work through long-term testing and cross-checking.</summary>
+    <updated>2025-08-14T11:15:01+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="tr">
+    <date form="text">
+      <date-part name="day" form="numeric" suffix=" "/>
+      <date-part name="month" suffix=" "/>
+      <date-part name="year"/>
+    </date>
+    <terms>
+      <term name="edition">basım</term>
+      <term name="anonymous"/>
+      <term name="curator" form="verb-short">thk. ve çev.</term>
+      <term name="issue" form="short"/>
+      <term name="compiler" form="verb-short">der.</term>
+      <term name="number" form="verb-short">Kanun No</term>
+      <term name="no date" form="short">t.y.</term>
+      <term name="et-al">vd.</term>
```

Comment:
Here' you are redefinding et-al to vd. even though the locale (translation) that comes with Zotero already does so for you. please remove any such duplications. See the tr-TR locale: https://github.com/citation-style-language/locales/blob/master/locales-tr-TR.xml